### ISSUE TITLE
NoSuchMethodError in Release 4.5.0

### ISSUE THREAD

**Issue by skohlmann** [victools/jsonschema-generator#42]
Hi,

again a NoSuchMethodError which breaks our nightly build with version range. 

----

 [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.071 s <<< FAILURE! - in schema.SchemaExportTest
369 20832 [ERROR] print  Time elapsed: 1.071 s  <<< ERROR!
370 java.lang.NoSuchMethodError: com.github.victools.jsonschema.module.jackson.JacksonModule: method <init>()V not found
371 	at schema.SchemaExportTest.lambda$print$0(SchemaExportTest.java:48)
372 	at schema.SchemaExportTest.print(SchemaExportTest.java:35)

----

Code:

```java
    public void createSchema(final Class clazz, final Writer target) throws IOException {
        final ObjectMapper objectMapper = new ObjectMapper();
        objectMapper.writerWithDefaultPrettyPrinter();
        final var configBuilder = new SchemaGeneratorConfigBuilder(objectMapper, OptionPreset.PLAIN_JSON);
        final var config = configBuilder.with(Option.NONPUBLIC_NONSTATIC_FIELDS_WITHOUT_GETTERS)
                                        .with(new JacksonModule())
                                        .with(new JavaxValidationModule(INCLUDE_PATTERN_EXPRESSIONS, NOT_NULLABLE_FIELD_IS_REQUIRED))
                                        .build();
        final var generator = new com.github.victools.jsonschema.generator.SchemaGenerator(config);
        final var jsonSchema = generator.generateSchema(clazz);

        target.write(jsonSchema.toPrettyString());
        target.flush();
    }
```

Runtime: Java 11

**Comment by CarstenWickner** [comment]
Hi @skohlmann,

Sorry about that.
I was really hoping it wouldn't be that strict.
I had replaced the `JacksonModule()` constructor with a varargs `JacksonModule(JacksonOption...)` constructor, which implictly also caters for `new JacksonModule()` to be called, i.e. shouldn't be a compilation issue.

I've added this (redudant) argument-free constructor again in [v4.5.1](https://github.com/victools/jsonschema-module-jackson/releases/tag/v4.5.1) of the `jsonschema-module-jackson`.

Thanks again for reporting!
